static/js/components/PhotometryPlot.jsx

```diff
@@ -1005,6 +1015,22 @@ const PhotometryPlot = ({
     }
   }, [showNonDetections]);
 
+  useEffect(() => {
```

Comment:
There is still a bug @theobonzi. If the datapoint is an upper limit with forced photometry origin, if the upper limits are deactivated and the forced photometry too, reactiving only the forced photometry should NOT show the upper limits again (since they are still deactivated).

See that example here:
<img width="831" alt="image" src="https://github.com/skyportal/skyportal/assets/49785117/9a623586-8327-438c-9b65-db1a8f92901f">